charts/trento-server/charts/trento-wanda/values.yaml

```diff
@@ -56,7 +56,7 @@ podSecurityContext:
 
 securityContext:
   {}
-  # capabilities:
+  # capabilitie  s:
```

Comment:
unwanted hunk, I guess

Reply:
Fixed in https://github.com/trento-project/helm-charts/pull/132/commits/8d09c1676399b00603ab4de6f849c03204016e28